aws/resource_aws_ses_domain_identity_verification.go

```diff
@@ -25,12 +24,10 @@ func resourceAwsSesDomainIdentityVerification() *schema.Resource {
 				Computed: true,
 			},
 			"domain": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
-				StateFunc: func(v interface{}) string {
-					return strings.TrimSuffix(v.(string), ".")
-				},
+				Type:      schema.TypeString,
+				Required:  true,
+				ForceNew:  true,
+				StateFunc: trimTrailingPeriod,
```

Comment:
Similar notes here about removing this (now) mostly extraneous handling in this resource. 👍